DTDL/company.json

```diff
@@ -0,0 +1,76 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:io:tributech:company;1",
+  "@type": "Interface",
+  "displayName": "Company",
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "City",
```

Comment:
thats what i wanted to do as well, however as company is used as a component in the node model we are not allowed to reference the address here as a component. (max depth of components is 1). We could do it via relationship but I did not like that as we should only do that when we create reusable entries in my opinion.

Reply:
Okay, I understand. One could argue that an address can be reused so a relationship would fit, but it's nothing that important.
But I think we can than remove `dtmi:io:tributech:address;1` since it's not used anywhere.